source/system/Demo_User_Guides/TI_Apps_Launcher_User_Guide.rst

```diff
@@ -365,14 +365,70 @@ _________________________
 
 3. Once you enter the necessary values for both HTTPS and No proxy, click on `Set Proxy` to set the proxy configuration.
 
+
+Using terminal
```

Comment:
Sections use ----, not ____. https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
Noted will update